charts/jenkins/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 
 The change log until v1.5.7 was auto-generated based on git commits. Those entries include a reference to the git commit to be able to get more details.
 
+## 2.8.2
+
+Fix indentation of JAVA_OPTS
```

Comment:
@holmesb I think this should be changed to "Added prometheus metric documentation."